library/general/src/modules/Linuxrc.rb

```diff
@@ -244,23 +244,28 @@ def value_for(feature_key)
     # Reset settings for vnc, ssh,... in install.inf which have been made
     # by linuxrc settings.
     #
-    # @param [Array<String>] list of services which will be disabled.
+    # @param [Array<String>] list of remote-management services that will be disabled.
     def disable_remote(services)
       return if !services || services.empty?
-      log.warn "Disabling #{services} due missing packages."
+
+      log.warn "Disabling #{services} due to missing packages."
       services.each do |service|
-        case service
+        # Service IDs are also used in another context in the code
+        # Making sure we always compare apples with apples
+        case polish(service.dup)
```

Comment:
The `dup` is not needed, the `downcase` in `key.downcase.tr("-_\\.", "")` (in the `polish` methos) already creates a copy of the string. See the documentation: https://ruby-doc.org/core-2.4.0/String.html#method-i-downcase

Reply:
this is actually not true. If service does not contain uppercase it is not copy. Try it yourself with

```
irb(main):001:0> a = "a"
=> "a"
irb(main):002:0> a === a.downcase
=> true
irb(main):003:0> a === a.upcase
=> false
```

Reply:
Oh, I see, that's quite tricky...